services/slam/builtin/orbslam_int_test.go

```diff
@@ -32,7 +32,6 @@ func createVocabularyFile(name string) error {
 }
 
 func TestOrbslamIntegrationExample(t *testing.T) {
-	// TODO DATA-364: remove this check
```

Comment:
I decided to leave this check in place, so that local tests that don't have an orbslam binary (e.g. on a mac) skip this test.

Reply:
This also allows the test to be skipped in the "Build and Test" job.